website/content/labs/lab5.md

```diff
@@ -183,9 +183,10 @@ As above, you should use the TDD workflow when implementing these new functions.
 
 Finally, as before, you should include header comments in all the tests you write.
 
-# Submitting your lab
+## Submitting your lab
+
+Before submitting, make sure you've added, committed, and pushed all your code to GitHub. Like the previous lab, you will submit your code through Gradescope,
```

Comment:
Maybe we could remind them specifically to make sure they've added any untracked files? Only because I think a couple people I graded in Lab 2 didn't include their new files.

Reply:
I've added a reminder, and also added a reminder in the TDD section to remind them to add the new circle files.